packages/server/lib/browsers/browser-cri-client.ts

```diff
@@ -67,7 +67,7 @@ const retryWithIncreasingDelay = async <T>(retryable: () => Promise<T>, browserN
 }
 
 export class BrowserCriClient {
-  private currentlyAttachedTarget: CRIWrapper.Client | undefined
+  private _currentlyAttachedTarget: CRIWrapper.Client | undefined
```

Comment:
I consider private values with public getters to be a JS anti-pattern - it adds code and verbosity for no good reason that I can tell. Very much like to avoid C# or Java getters and setters in our code.

(enforcing access control is not, IMO, a good reason)